dags/atd_executive_dashboard_row_active_permits_logging.py

```diff
@@ -114,4 +150,16 @@
         retry_delay=duration(seconds=60),
     )
 
-    t1
+    t2 = DockerOperator(
+        task_id="backup_active_permits",
+        image=docker_image_2,
+        auto_remove=True,
```

Comment:
We have an open issue to do this, but I've been adding the default docker hub connection to my new dags

https://github.com/cityofaustin/atd-airflow/pull/165/files#diff-6c336b555e2faf578eda218ff84472fe28d1c7787414c68a8168e0193a13eac5R95